Lib/test/test_httpservers.py

```diff
@@ -17,8 +17,11 @@
 import urllib.parse
 import html
 import http.client
+import email.message
```

Comment:
Just place `import email.utils` the first and `import datetime` after it. You can also move `import base64` before them.